driver.sh

```diff
@@ -81,12 +82,13 @@ check_dependencies() {
   command -v timeout
   command -v unbuffer
   command -v clang-8
+  command -v "${LD:="${CROSS_COMPILE:-}"ld}"
 
   set +e
 }
 
 mako_reactor() {
-  make -j"$(nproc)" CC="${ccache} ${clang}" HOSTCC="${ccache} ${clang}" "${@}"
+  make -j"$(nproc)" CC="${ccache} ${clang}" HOSTCC="${ccache} ${clang}" LD="${LD}" ${@}"
```

Comment:
Whoops, a quotation mark got lost it seems.

Reply:
back to first grade for me! :ice_cream: :horse_racing: